views/about_me.erb

```diff
@@ -0,0 +1,74 @@
+<html>
+<head>
+<title>Lauren Granger's Awesome Website of Awesomeness</title>
+</head>
+
+<body>
+  <header>
+    <h1>Welcome to Lauren's Awesome Website of Awesomeness</h1>
+  <nav>
+    <ul>
+      <strong><li><a href="index">Home</a></li></strong>
+      <li><a href="about_me">All About Lauren</a></li>
+      <li><a href="blog">Blog</a></li>
+      <li><a href="projects">Projects</a></li>
+    </ul
+  </nav>
+  <img src="./images/break_line.png"/>
+  </header>
+
+  <table style="text-align: center">
+    <thead>
+      <th colspan="3"><h1>
+        Hello World! I'm Lauren!
+      </h1></th>
```

Comment:
I think it would make sense to either have the opening and closing `h1` tag on line 23, or have them on separate lines completely so that the `th` are on separate lines as well